### PR TITLE
Specify location

### PR DESCRIPTION
**Pull request by kaadhina**
Fixes

### CODE REVIEW

File: Tasks/VsTest/helpers.ts [hunk 1]
```diff
@@ -12,6 +12,11 @@ var xml2js = require('xml2js');
 var parser = new xml2js.Parser();
 var builder = new xml2js.Builder();
 
+export class Constants {
+    public static vsTestVersionString = 'version';
```
Comment (nigurr) [inline Tasks/VsTest/helpers.ts]: constants along with helper functions?

File: Tasks/VsTest/versionfinder.ts [hunk 2]
```diff
@@ -22,21 +23,41 @@ export function locateVSTestConsole(testConfig): Q.Promise<models.ExecutabaleInf
...
-    if (isNaN(vsVersion) || vsVersion === 15.0) {
-        // latest
-        tl.debug('Searching for latest Visual Studio');
-        let vstestconsole15Path = getVSTestConsole15Path(testConfig.vs15HelperPath);
-        if (vstestconsole15Path) {
-            deferred.resolve({ version: 15.0, location: vstestconsole15Path });
+    if(testConfig.vsTestLocationMethod === utils.Constants.vsTestLocationString) {
+        if (utils.Helper.pathExistsAsFile(testConfig.vsTestLocation)) {
+            deferred.resolve({ version: null, location: path.join(testConfig.vsTestLocation,"..")});
         } else {
```
Comment (nigurr) [inline Tasks/VsTest/versionfinder.ts]: why null? what if there's dependency on this down the line?

Comment (nigurr) [inline Tasks/VsTest/versionfinder.ts]: else if

**Comment by nigurr** [comment]
:shipit:

File: Tasks/VsTest/helpers.ts (hunk 1 above)
Comment (kaadhina) [inline Tasks/VsTest/helpers.ts]: Will refactor to a contants file when more constants are added

File: Tasks/VsTest/versionfinder.ts (hunk 2 above)
Comment (kaadhina) [inline Tasks/VsTest/versionfinder.ts]: for specify location scenario we dont need the version

**Pull request merged**